stage/dotnet/dotnet-sdk-enterprise-cloud/test-pr.txt

```diff
@@ -0,0 +1 @@
+test
```

Comment:
What is this file? Does this need to be here?

Reply:
One that needs to be deleted... thx for catching that.